BackEnd/src/main/java/com/unosquare/admin_core/back_end/configuration/mappings/domain/EventMappings.java

```diff
@@ -17,6 +17,7 @@ protected void configure() {
                 map().setEventType(new EventType(source.getEventTypeId()));
                 map().setStartDate(source.getStartDate());
                 map().setEndDate(source.getEndDate());
+                map().setDeniedMessage(source.getDeniedMessage());
```

Comment:
I don't think we should couple the message to 'denied'. something like comment or message. That way it's property isn't just locked down to one event activity.